include/swift/AST/DiagnosticsSema.def

```diff
@@ -2533,8 +2533,11 @@ NOTE(silence_inject_forced_downcast,none,
      "add parentheses around the cast to silence this warning", ())
 
 ERROR(conditional_downcast_foreign,none,
-      "conditional downcast to CoreFoundation type %0 will always succeed",
+     "conditional downcast to CoreFoundation type %0 will always succeed",
       (Type))
+NOTE(compare_CFTyteId_foreign,none,
```

Comment:
Nit: Could you fixup the selector’s name?

Reply:
👌